dask_distance/__init__.py

```diff
@@ -365,6 +365,44 @@ def minkowski(u, v, p):
     return result
 
 
+def seuclidean(u, v, V):
+    """
+    Finds the standardized Euclidean distance between two 1-D arrays.
+
+    .. math::
+
+       \sqrt{\sum_{i} \left( \\frac{u_{i}^{2} - v_{i}^{2}}{V_{i}} \\right)}
```

Comment:
This equation is incorrect. The difference should be squared instead of taking the difference of the squares. Fixing in PR ( https://github.com/jakirkham/dask-distance/pull/59 ).